doc/man/man3/crypto_poly1305.3monocypher

```diff
@@ -173,23 +171,18 @@ Take the first 32 bits of that stream as your authentication key, then
 use the
 .Em rest
 of the stream to encrypt your message.
-The source code of
-.Xr crypto_aead_lock 3monocypher
-shows how it is done.
+This is the approach used by
+.Xr crypto_aead_lock 3monocypher .
 .Ss Protection against side channels
 Use
 .Xr crypto_verify16 3monocypher
-To compare message authentication code.
-Avoid standard buffer comparison functions.
-They do not run in constant time, which allows attackers to recover
-the MAC in relatively few tries.
+to compare message authentication codes.
+Avoid standard buffer comparison functions, as they may not run in
+constant time.
```

Comment:
I agree with the change, but I wonder if we want to add a little additional warning: the problem with standard comparison functions is that timing attacks are not only possible, they are almost *easy*, even over the network. Other algorithms (Argon2d, naive AES, scrypt…) don't run in constant time, yet remain harder to attack.

Maybe add something like "This allows timing attacks" or something? I know it's redundant, and I don't like the ugliness, but I think this is worth emphasising.

Reply:
yeah I guess if the manual has warnings it should explain them rather than leaving them as mysteries